app/Livewire/Project/Shared/ResourceLimits.php

```
@@ -11,52 +11,105 @@ class ResourceLimits extends Component
+    // Explicit properties
+    public ?string $limitsCpus = null;
+
+    public ?string $limitsCpuset = null;
+
+    public ?int $limitsCpuShares = null;
+
+    public string $limitsMemory;
+
+    public string $limitsMemorySwap;
+
+    public int $limitsMemorySwappiness;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $limitsMemorySwappiness. Keep variable name length under 20. (undefined)

(LongVariable)